plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/TrinoCatalog.java

```diff
@@ -102,6 +109,21 @@ Transaction newCreateTableTransaction(
             String location,
             Map<String, String> properties);
 
+    default void validateCreateTableTransaction(String tableLocation, TrinoFileSystem fileSystem)
```

Comment:
This could go into `IcebergUtil` instead of being in `TrinoCatalog`, right? Now that individual catalogs are calling this method within their `newCreateTrableTransaction` method, it doesn't need to be an interface.

Reply:
Moved to `IcebergUtil`